data/tlg0008/tlg001/tlg0008.tlg001.perseus-grc4.xml

```diff
@@ -1,4 +1,5 @@
-<?xml version="1.0" encoding="UTF-8"?><TEI xmlns="http://www.tei-c.org/ns/1.0">
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
For this file but also the others, you are missing the Epidoc declaration. It is not tested (partly because I do not know how to)
```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="http://www.stoa.org/epidoc/schema/8.19/tei-epidoc.rng"
  schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="http://www.stoa.org/epidoc/schema/8.19/tei-epidoc.rng"
  schematypens="http://purl.oclc.org/dsdl/schematron"?>
```